Dockerfile

```
@@ -0,0 +1,8 @@
+FROM registry.olxbr.io:5000/python:3.7-slim
```

Depois temos que mover isso para usar o ECR.

cc/ @olxbr/squad-cross-dev-tools

---

@marciorgb acha isso um blocker pra aceitar o MR? Caso seja, como faço pra adicionar o python slim no nosso ECR?

---

Subi a imagem python:3.7-slim do dockerhub para um repo público da conta cross olx:

```suggestion
FROM public.ecr.aws/o5c8q0l3/python:3.7-slim
```